contracts/common/misc/DrainStakeManager.sol

```diff
@@ -0,0 +1,12 @@
+pragma solidity ^0.5.2;
+
+import { StakeManagerStorage } from "../../staking/stakeManager/StakeManagerStorage.sol";
+import { Lockable } from "../mixin/Lockable.sol";
+
+contract DrainStakeManager is StakeManagerStorage {
+    constructor() public Lockable(address(0x0)) {}
+
+    function drain(address destination, uint amount) external onlyGovernance {
+        require(token.transfer(destination, amount));
+    }
+}
```

Comment:
Please add an empty line at the end of the file. You can set it automatically in vs code: https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files

Reply:
The reasoning behind it: https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file